checkout/module.go

```diff
@@ -190,3 +191,27 @@ func (m *Module) Depends() []dingo.Module {
 		new(flamingo.SessionModule),
 	}
 }
+
+type apiRoutes struct {
+	apiController *controller.APIController
+}
+
+func (r *apiRoutes) Inject(apiController *controller.APIController) {
+	r.apiController = apiController
+}
+
+func (r *apiRoutes) Routes(registry *web.RouterRegistry) {
+	registry.Route("/api/v1/checkout/placeorder", "checkout.api.placeorder")
+	registry.HandleGet("checkout.api.placeorder", r.apiController.CurrentPlaceOrderContextAction)
+	registry.HandlePut("checkout.api.placeorder", r.apiController.StartPlaceOrderAction)
+	registry.HandleDelete("checkout.api.placeorder", r.apiController.ClearPlaceOrderAction)
+
+	registry.Route("/api/v1/checkout/placeorder/cancel", "checkout.api.placeorder.cancel")
+	registry.HandleGet("checkout.api.placeorder.cancel", r.apiController.CancelPlaceOrderAction)
+
+	registry.Route("/api/v1/checkout/placeorder/refreshplaceorder", "checkout.api.placeorder.refreshplaceorder")
```

Comment:
why duplication of placeorder?
```suggestion
	registry.Route("/api/v1/checkout/placeorder/refresh", "checkout.api.placeorder.refresh")
```